source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -38,7 +38,8 @@ AndroidContext: class extends OpenGLContext {
 	}
 	createGpuImage: override func (rasterImage: RasterImage) -> GpuImage {
 		result: GpuImage
-		if (rasterImage instanceOf?(GraphicBufferYuv420Semiplanar)) {
+		//This optimization is currently disabled since it's only faster for 4k resolution
+		if (false && rasterImage instanceOf?(GraphicBufferYuv420Semiplanar)) {
```

Comment:
Can we do something less confusing than `if (false` here? How about a version flag or a setting, somewhere?